### PR TITLE
Move results to page content, change works url

### PR DESCRIPTION
**Pull request by rcantin-w**
## Who is this for?
New search page 

## What is it doing for them?
As discussed, moving # results to the page content itself
Change Catalogue's URL back ("Catalogue" is label, `works` is URL)

### CODE REVIEW

File: catalogue/webapp/pages/search/images.tsx [hunk 1]
```diff
@@ -57,6 +58,8 @@ const ImagesSearchPage: NextPageWithLayout<Props> = ({
     setLink(link);
   }, [imagesRouteProps]);
 
+  const showSort = true;
```
Comment (gestchild) [inline catalogue/webapp/pages/search/images.tsx:61]: is this going to be set some other way in the future?

**Review by gestchild — APPROVED** [review]

File: catalogue/webapp/pages/search/images.tsx (hunk 1 above)
Comment (AMBacelar) [reply catalogue/webapp/pages/search/images.tsx:61]: That is removed in one of my following PRs

Comment (rcantin-w) [reply catalogue/webapp/pages/search/images.tsx:61]: @AMBacelar am I OK to merge it?

Comment (AMBacelar) [reply catalogue/webapp/pages/search/images.tsx:61]: Indeed, I'll have that resolved in the sort PR

**Pull request merged**